components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -215,6 +215,12 @@ export const GridItemBrandedLogo = styled(GridItemCredits)`
   --ntp-grid-item-credits-left-margin-narrow: 0;
   --ntp-grid-item-credits-bottom-margin-wide: -8px;
   --ntp-grid-item-credits-left-margin-wide: 22px;
+  
+  @media screen and (min-width: ${breakpointEveryBlock + 1}) {
```

Comment:
Just wondering why we're looking at viewport width and not height?

Reply:
This isn't actually doing anything - browser is saying invalid css. That's because `breakpointEveryBlock` is a string (`'870px'`) so this results in string `'870px1'`. We'd want to do something like 
```
@media screen and (min-width: calc(${breakpointEveryBlock} + 1))
```

Reply:
Wow... sorry for missing that. I should have tested that better. Pushing a chance shortly.

Reply:
`calc` isn't working in the media query, so I'm sticking with the raw `breakpointEveryBlock` which results in what's pictured below at 870px.
<img width="1053" alt="Screen Shot 2021-01-15 at 8 02 33 PM" src="https://user-images.githubusercontent.com/1199820/104792531-afa76600-576c-11eb-97fe-c44ed1c3f705.png">